references/video_classification/README.md

```diff
@@ -0,0 +1,33 @@
+# Video Classification
+
+TODO: Add some info about the context, dataset we use etc
+
+## Data preparation
+
+If you already have downloaded [Kinetics400 dataset](https://deepmind.com/research/open-source/kinetics), 
+please proceed directly to the next section.
+
+To download videos, one can use https://github.com/Showmax/kinetics-downloader
+
+## Training
+
+We assume the training and validation AVI videos are stored at `/data/kinectics400/train` and 
+`/data/kinectics400/val`. 
+
+### Multiple GPUs
+
+Run the training on a single node with 8 GPUs:
```

Comment:
I believe @bjuncek trained those models on 64 GPUs, it might be good to add a mention here in a follow-up PR